enacts/crop_suitability/layout_crop_suit.py

```diff
@@ -0,0 +1,374 @@
+import os
+from dash import dcc
+from dash import html
+import dash_bootstrap_components as dbc
+from dash import dash_table
+import dash_leaflet as dlf
+import plotly.express as px
+from controls import Block, Sentence, DateNoYear, Number
+
+import numpy as np
+from pathlib import Path
+import calc
+import pingrid
+import pandas as pd
+
+
+GLOBAL_CONFIG = pingrid.load_config(os.environ["CONFIG"])
+CONFIG = GLOBAL_CONFIG["crop_suit"]
+
+DR_PATH = GLOBAL_CONFIG["rr_mrg_zarr_path"]
```

Comment:
Yes I am not able to run the app at present because of this. But @aaron-kaplan , I am also not able to run the other app (wat_bal) that is not part of the multi-page I think because I don't know how to make it know globals_....